src/reactor.h

```diff
@@ -0,0 +1,342 @@
+    "tooltip": "Reactor is active for 1 timestep, then dormant for buy_frequency-1 timesteps", \
+    "units": "Timesteps", \
+    "uitype": "range", \
+    "range": [0, 1e299], \
+    "uilabel": "Buy frequency" \
+  }
+  int buy_frequency;
+
+  #pragma cyclus var { \
+    "doc": "Fresh fuel commodity", \
+    "tooltip": "Name of fuel commodity requested", \
+    "uilabel": "Fuel input commodity" \
+  }
+  std::string fuel_incommod;
+
+  #pragma cyclus var { \
+    "doc": "Fresh fuel commodity", \
+    "tooltip": "Name of fuel commodity requested", \
+    "uilabel": "Fuel input commodity" \
+  }
+  std::string he3_outcommod;
+
+  #pragma cyclus var { \
+    "default": 0.03, \
+    "doc": "Fraction of tritium that comes from the (n + Li-7 --> T + He + n) reaction", \
+    "tooltip": "Fraction of tritium from Li-7 breeding", \
+    "units": "dimensionless", \
+    "uitype": "range", \
+    "range": [0, 1], \
+    "uilabel": "Li-7 Contribution" \
+  }
+  double Li7_contribution;
+
+  #pragma cyclus var { \
+    "default": 1000.0, \
+    "doc": "Initial mass of full blanket material", \
+    "tooltip": "Only blanket material mass, not structural mass", \
+    "uitype": "range", \
+    "range": [0, 10000], \
+    "uilabel": "Initial Mass of Blanket" \
+  }
+  double blanket_size;
+
+  #pragma cyclus var { \
+    "doc": "Fresh fuel commodity", \
+    "tooltip": "Name of fuel commodity requested", \
+    "uilabel": "Fuel input commodity" \
+  }
+  std::string blanket_incommod;
+
+  #pragma cyclus var { \
+    "doc": "Fresh fuel commodity", \
+    "tooltip": "Name of fuel commodity requested", \
+    "uilabel": "Fuel input commodity" \
+  }
+  std::string blanket_inrecipe;
+
+  //WARNING: The default on this is completely arbitrary!
+  #pragma cyclus var { \
+    "default": 0.05, \
+    "doc": "Percent of blanket that gets recycled every blanket turnover period", \
+    "tooltip": "Defaults to 0.05 (5%), must be between 0 and 15%", \
+    "units": "dimensionless", \
+    "uitype": "range", \
+    "range": [0, 0.15], \
+    "uilabel": "Blanket Turnover Rate" \
+  }
+  double blanket_turnover_rate;
+
+  #pragma cyclus var { \
+    "default": 1, \
+    "doc": "number of timesteps between blanket recycles", \
+    "tooltip": "Defaults to 0.05 (5%), must be between 0 and 15%", \
+    "units": "dimensionless", \
+    "uitype": "range", \
+    "range": [0, 1000], \
+    "uilabel": "Blanket Turnover Rate" \
+  }
+  int blanket_turnover_frequency;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling tritium material to be used in reactor"}
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_storage;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling excess tritium material to be sold"}
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_excess;
+
+  #pragma cyclus var {"tooltip":"Buffer for handling tritium sequestered in the system"}
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_sequestered;
+
```

Comment:
These can just be `Material` - will make your life easier, I think.

Reply:
Changed sequestered_tritium to a material. tritium_excess needs to be a buffer since it's how we sell off excess tritium (there's a policy attached). Checked all the other buffers to see if they could be converted, and the only other one was blanket_excess, which should eventually have a policy added (in a future version), so I'm going to leave that one as a buffer for now.